recipes/proposal/all/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
+project(cmake_wrapper LANGUAGES C)
```

```suggestion
project(cmake_wrapper)
```

It's not a C project